drivers/video/video.c

```diff
@@ -1236,13 +1307,27 @@ static int video_qbuf(FAR struct video_mng_s *vmng,
                       (&type_inf->bufinf);
           if (container)
             {
-              start_capture(buf->type,
-                            type_inf->nr_fmt,
-                            type_inf->fmt,
-                            &type_inf->clip,
-                            &type_inf->frame_interval,
-                            container->buf.m.userptr,
-                            container->buf.length);
+              if (container->buf.memory == V4L2_MEMORY_MMAP)
```

Comment:
why do you need check V4L2_MEMORY_MMAP? userptr already point to the right place.

Reply:
userptr in MMAP case is a union shared with offset, so it is not pointing to the right place?